docs/resources/azure_data_lake_storage_gen2_path.md

```diff
@@ -0,0 +1,106 @@
+---
+title: About the azure_data_lake_storage_gen2_path Resource
+platform: azure
+---
+
+# azure_data_lake_storage_gen2_path
+
+Use the `azure_data_lake_storage_gen2_path` InSpec audit resource to test the properties related to Azure Data Lake Storage Gen2 Filesystem.
+
+## Azure REST API version, Endpoint, and HTTP Client Parameters
+
+This resource interacts with API versions supported by the resource provider. The `api_version` is defined as a resource parameter.
+If not provided, the latest version is used. For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint and default values for the HTTP client are used. For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). For an example, `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`name`, `account_name` and `path` is a required parameter and `dns_suffix` is an optional parameter.
+
+```ruby
+describe azure_data_lake_storage_gen2_path(account_name: 'ACCOUNT_NAME', filesystem: 'FILE_SYSTEM', name: 'PATHNAME') do
```

Comment:
There's a mismatch between the parameters here and ones listed in below in the parameters table.

Reply:
updated docs!